File: LeetCode/Java Solutions/Three Consecutive Odds/three.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        for (int i = 0; i < arr.length - 2; i++) {
            if (arr[i] % 2 != 0 && arr[i + 1] % 2 != 0 && arr[i + 2] % 2 != 0) {
                return true;
            }
        }
        return false;
    }

    public static void main(String[] args) {
        Solution sol = new Solution();
        System.out.println(sol.threeConsecutiveOdds(new int[]{2, 6, 4, 1}));  // Output: false
        System.out.println(sol.threeConsecutiveOdds(new int[]{1, 2, 34, 3, 4, 5, 7, 23, 12}));  // Output: true
    }
}

File: LeetCode/Python Solutions/Three Consecutive Odds/three.py
from typing import List

class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        for i in range(len(arr) - 2):
            if arr[i] % 2 != 0 and arr[i + 1] % 2 != 0 and arr[i + 2] % 2 != 0:
                return True
        return False

# Example usage:
sol = Solution()
print(sol.threeConsecutiveOdds([2, 6, 4, 1]))  # Output: False
print(sol.threeConsecutiveOdds([1, 2, 34, 3, 4, 5, 7, 23, 12]))  # Output: True
